webapp/src/components/Controls/Controls.tsx

```diff
@@ -207,6 +208,9 @@ const Controls: React.FC<Props> = ({
           <Box display="flex" alignItems="center" gap={1} whiteSpace="nowrap">
             <TuneIcon />
             <Typography variant="subtitle2">Controls</Typography>
+            <Box paddingTop={0.5}>
+              <Description link="/exploration-space/#filters" />
+            </Box>
```

Comment:
With my previous suggestion, that should align perfectly without tweaking it:
```suggestion
            <Description link="/exploration-space/#filters" />
```

Reply:
ohh yes right..